client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/InvestigationTableFooterInterfaces.ts

```diff
@@ -20,4 +20,5 @@ export interface InvestigationTableFooterOutcome {
     handleCloseGroupedInvestigations: () => void;
     handleConfirmDesksDialog: (updatedDesk: Desk, transferReason: string) => void;
     handleConfirmInvestigatorsDialog: (updatedIvestigator: InvestigatorOption, transferReason: string) => void;
+    handleDispandGroupedInvestigations: (groupIds: string[]) => void;
```

Comment:
handleDispandGroupedInvestigations -  did you mean handle**Disband**GroupedInvestigations ?

Reply:
Oops

Reply:
fixed